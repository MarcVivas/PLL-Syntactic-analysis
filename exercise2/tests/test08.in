a = 1;
b = 2;
a < b ? a : b;
a > b ? a : b;
a == b ? a : b;
a != b ? a+2 : b-4;
a <= b ? a+1 : b+2;
a >= b ? a+74 : b+4;
a < b ? (a == 1 ? a : b) : b;
a > b ? (b == 2 ? a : b) : a;
a <= b ? (a != b ? a : b) : b;
a >= b ? (b == 2 ? a : (a != 1 ? b : a)) : a;
a+b >= b ? c = 4234-11 : c = 55-1*4+3-10/2;
c;
a-b >= b ? c = 6465465-11 : c + 4;
a%b >= b ? c*78 : c = -4;
c;