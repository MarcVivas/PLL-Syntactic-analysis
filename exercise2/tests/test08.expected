1
2
2
3
2
6
1
1
1
1
4223
4227
-4
a = 1
b = 2
c = -4